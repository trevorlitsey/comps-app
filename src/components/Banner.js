import React from 'react'

const Banner = props =>
	<div className="container__banner margin-auto width-420">
		<h2 className="lobster">{props.text}</h2>
	</div>

export default Banner;