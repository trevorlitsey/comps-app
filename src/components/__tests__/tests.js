import React from 'react';
import Banner from '../Banner';

import renderer from 'react-test-renderer';

it('sums numbers', () => {
	expect(1 + 2).toEqual(3);
});

it('renders link', () => {
	const banner = renderer
		.create(<Banner />)
		.toJSON();
	expect(banner).toMatchSnapshot();
})