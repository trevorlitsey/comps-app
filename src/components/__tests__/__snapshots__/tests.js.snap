// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders link 1`] = `
<div
  className="container__banner margin-auto width-420"
>
  <h2
    className="lobster"
  />
</div>
`;
